build.gradle

```diff
@@ -499,6 +512,11 @@ task api(type: Javadoc) {
 	destinationDir = new File(buildDir, 'api')
 }
 
+tasks.withType(Javadoc) {
+	options.addBooleanOption('Xdoclint:syntax', true) // only check syntax with doclint
+	options.addBooleanOption('Werror', true) // fail build on Javadoc warnings
+}
```

Comment:
This block has to be moved to the `subprojects` section.
Turns out we build javadoc jar from every single module.
And then we also build a single consolidated javadoc.
Don't ask me why we do that: probably SF does the same.
Or they just don't publish javadoc jars...